webapp/src/ts/services/task-notifications.service.ts

```diff
@@ -0,0 +1,103 @@
+import { Injectable } from '@angular/core';
+import * as moment from 'moment';
+import { orderBy } from 'lodash-es';
+
+import { RulesEngineService } from '@mm-services/rules-engine.service';
+import { TranslateService } from '@mm-services/translate.service';
+import { DBSyncService } from '@mm-services/db-sync.service';
+
+/* 
+** avoid overloading app with too many notifications especially at once
+** 24 for android >= 10
+*/
+const MAX_NOTIFICATIONS = 24;
```

Comment:
I'm curious how this number was selected. This seems like an incredibly high number. I believe we should consult with some UI experienced designers for the number of notifications that is acceptable, and doesn;t become overwhelming. 
I believe we should also make this configurable.

Reply:
- Starting with android 10 android dropped [max number of notifications](https://support.goo
gle.com/pixelphone/thread/23619509/notification-limits-dropped-to-24-in-android-10-can-i-change-that?hl=en) to 24. Anything after 24 is discarded. This is what i went off of
- for the android UI if notifications are more than 4 they'll be grouped like below

Grouped by default
![Screenshot_20250610_095750_One UI Home](https://github.com/user-attachments/assets/19004af3-88ad-4d24-b8cb-7ff93777ae76)

when user expands them
![Screenshot_20250610_095801_One UI Home](https://github.com/user-attachments/assets/525e76ce-e667-4eb8-8446-f730e30be07f)

LMK what you think especially on the making this number configurable